worker-core/src/main/java/com/hpe/caf/worker/core/StreamingWorkerWrapper.java

```diff
@@ -53,11 +55,13 @@ public StreamingWorkerWrapper(final WorkerTaskImpl workerTask)
     @Override
     public void run()
     {
+        final String workerFriendlyName = !Strings.isNullOrEmpty(System.getenv(CAF_WORKER_NAME)) ?
+                System.getenv(CAF_WORKER_NAME) : worker.getClass().getName();
         try {
             if (workerTask.isPoison()) {
-                LOG.warn("Worker [" + worker.getWorkerIdentifier() + "] did not handle poisoned message, when it was passed for processing.");
+                LOG.warn(workerFriendlyName + " could not process the document.");
```

Comment:
Sorry, in hindsight changing this to refer to a document is wrong within the worker framework, it would be fine to use the term Document in the document worker framework.

Let's go for "could not process the item."